server/app/src/androidTest/java/sh/calaba/instrumentationbackend/query/ast/UIQueryASTWith.java

```diff
@@ -289,6 +291,18 @@ private boolean isMarked(Object o, Object expectedValue) {
 			return true;
 		}
 
+		// Workaround for Android 2.3.x: retrieve text and hint manually to resolve issue with AppCompat elements.
```

Comment:
extend comment to explain issue with AppCompatTextView

Reply:
Ok, will do